libs/sdk-core/src/persist/migrations.rs

```diff
@@ -180,6 +180,7 @@ pub(crate) fn current_migrations() -> Vec<&'static str> {
        ",
        "ALTER TABLE payments_external_info ADD COLUMN ln_address TEXT;",
        "ALTER TABLE payments_external_info ADD COLUMN lnurl_metadata TEXT;",
+       "ALTER TABLE payments_external_info ADD COLUMN external_metadata TEXT;",
```

Comment:
We cannot insert or edit migrations defined already in the `current_migrations` / `current_sync_migrations`, as these migrations have already been run on existing clients using the SDK. Any migration needed has to be added to the end of the respective list.

So this line needs to be moved to the end of the `current_sync_migrations` list of migrations and you can remove the other changes here.

Reply:
Understood, thanks Ross